.github/workflows/run-qit.yml

```diff
@@ -0,0 +1,66 @@
+name: Run QIT
+
+# **What it does**: Runs a suite of QIT tests for the extension.
+# **Why we have it**: To be able to check QIT compatibility at once. For example, to test a specific branch, or upcoming release.
+
+on:
+  workflow_dispatch:
+    inputs:
+      # Configure which tests to run.
+      test-activation:
+        description: 'Should activation be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-security:
+        description: 'Should security be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-phpstan:
+        description: 'Should phpstan be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-api:
+        description: 'Should API be tested?'
+        required: true
+        default: true
+        type: boolean
+      test-e2e:
+        description: 'Should E2E be tested? (takes a lot of time)'
+        required: true
+        default: false
+        type: boolean
+
+      # Advanced customization.
+      options:
+        description: 'Additional options for `qit` command, like `--optional_features=hpos`.'
+        required: false
+
+  # Used to test this PR, to be removed after before merging.
+  push:
+    branches:
+      - dev/qit-workflow
+
+jobs:
+  build:
+    name: Build extension
+    uses: ./.github/workflows/build.yml
+    secrets: inherit
+  qit-tests:
+    name: Run QIT Tests
+    # Update it with more stable path once merged.
+    uses: woocommerce/grow/.github/workflows/run-qit-extension.yml@add/qit-workflows
+    needs: build
+    secrets: inherit
+    with:
+      version: local
+      # Conditional statements are here to allow testing on push triggers, without manual input. To be removed before merging.
+      test-activation: ${{ inputs.test-activation || true }}
+      test-security: ${{ inputs.test-security || true }}
+      test-phpstan: ${{ inputs.test-phpstan || true }}
+      test-api: ${{ inputs.test-api || false }}
```

Comment:
Test API is true by default in the inputs. Shouldn't be this consistent with it, as the rest of the parameters?